google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -164,6 +176,9 @@ def visit_BINARY(self, type_, **kw):
     def visit_DECIMAL(self, type_, **kw):
         return "NUMERIC"
 
+    def visit_NUMERIC(self, type_, **kw):
+        return "NUMERIC"
```

Comment:
With this change NUMERIC type will be created without precision and scale mentions. Even if user will set something with explicit dimensions, for example, `types.Numeric(precision=2, scale=2)`, in DDL only keyword `NUMERIC` will be used. Otherwise syntax error will fail.